Makefile

```diff
@@ -1,6 +1,7 @@
 include .env
 
 DB_DSN="${DB_USERNAME}:${DB_PASSWORD}@tcp(${DB_HOST}:${DB_PORT})/${DB_NAME}"
+BB_DSN_GEN = "${DB_USERNAME}:${DB_PASSWORD}@tcp(localhost:33066)/${DB_NAME}"
```

Comment:
it dont think this changes was necessary since host and port are loaded from .env

Reply:
@sawal-timsina 
ah however the db host wont be reccognized  by the host machine since the `gentool` has been installed into host machine. and the `database` service is running on docker machine 
so to access the tcp connect we need localhost and host exposed port which is 33066 instead which is being mapped into docker compose file. 
https://github.com/readytowork-org/go-gin-skeleton/blob/develop/docker-compose.yml#L46

Reply:
Alright, but let's configure the command to use DSN based on env provided